### ISSUE TITLE
Consider a mechanism such as "play order" for Fixed Layout publications

### ISSUE THREAD

**Issue by GeorgeKerscher** [mattgarrish/epub-revision#2664]
In the DAISY 3 Standard, ANSI/NISO Z39.86,  we had to resolve a problem with the proper sequence of content. I believe there is a similar problem with fixed layout publications.

The idea would be to establish a structure that identifies the correct sequence to present the information. A few could be chosen that presents an alternative to something in the fixed layout that is an equivalent representation of that portion of the fixed layout version. It should then be easy to toggle from the fixed layout view to the alternative version and maintain the position in both views.

From the DAISY 3 Standard we have:

8.4.3 playOrder Attribute
(This section is normative.)
The 
playOrder attribute is required on each 
pageTarget, 
navTarget and 
navPoint. It provides a means to collate all 
pageTargets, 
navTargets, and 
navPoints into a single ordered sequence that reflects their order in the normal playback sequence of the book as presented in the 
spine and SMIL file(s). 
playOrder is a positive integer; the first 
playOrder value in a document shall be 1. When the 
content elements of any 
pageTargets, 
navTargets, or 
navPoints reference the same SMIL time container, they must have the same 
playOrder value. 
playOrder must increase by one for each unique SMIL time container referenced by any 
pageTarget, 
navTarget or 
navPoint.## Issue on the [EPUB 3.3](https://www.w3.org/TR/epub-33/) Recommendation

**Comment by gregoriopellegrino** [comment]
Interesting solution to explore.

I think in the context of Fixed Layouts done with HTML and CSS, perhaps DOM order and CSS (z-index) can suffice to fix the visual presentation.

The challenge is when the reading order goes from one page to the one next to it and then back to the previous page... In this case defining a read order above the DOM order would make sense.

I think we should find a solution that works natively with browser rendering engines, otherwise I see complicated integration with assistive technologies and reading applications.

**Comment by mattgarrish** [comment]
> I think we should find a solution that works natively with browser rendering engines

The closest we have is `aria-flowto`, but it's limited to elements within the current document. We might want to work with that group to allow cross-document references, or to define something like an `aria-flowacross` for this case.

The web doesn't have a concept of spreads, so reading across two visible viewports is not a use case I expect they'd have considered.

**Comment by iherman** [comment]
This was discussed during the [pmwg meeting on 25 September 2025](https://w3c.github.io/pm-wg/minutes/2025-09-25.html#bb83).

<details><summary><i>View the transcript</i></summary>

<h3>Issue <a href="https://github.com/w3c/epub-specs/issues/2664" rel="noopener noreferrer">#2664</a> - <a href="https://github.com/w3c/epub-specs/issues/2664" rel="noopener noreferrer">w3c/<wbr>epub-specs#2664</a></h3>
<p><cite>wendreid:</cite> we don't have playorder officially from documents since we got rid of the NCX<br>
<span>… playorder is inferred from the spine</span></p>
<p><cite>duga:</cite> I assumed this was to put play order inside elements within the document?</p>
<p><cite>wendreid:</cite> are they talking about reading order within the HTML itself, that comes from the DOM</p>
<p><cite>mgarrish:</cite> without George here we aren't sure</p>
<p><cite>gautierchomel:</cite> maybe we can tag it with fixed layout<br>
<span>… in my experience in training, we have never had a problem with at fixed layout reading order</span><br>
<span>… it is doable for now</span><br>
<span>… let's tag it for fxl and discuss it in the fxl a11y context</span></p>
<p><cite>wendyreid:</cite> yes, let's do that</p>
<p><cite>gautierchomel:</cite> if you have two html files, and the title is part on one page, and part on the other, you may need to do this<br>
<span>… but it can be achieved with code now</span></p>
<p><cite>mgarrish:</cite> this sounds like rendition mapping<br>
<span>… he wants to move from fxl to something else</span><br>
<span>… which is a huge issue we've never solved</span><br>
<span>… and he wants a new mechanism for moving around within content</span></p>
<p><cite>wendyreid:</cite> I'll leave this alone until we can talk to George<br>
<span>… are there any more things to discuss</span></p>
<p><cite>gautierchomel:</cite> some of the issue topics can be pushed into discussion</p>
<p><cite>wendyreid:</cite> we didn't consider that, we didn't have that option in Github before<br>
<span>… we have used issues for everything. Now that we do have discussions as an option</span><br>
<span>… we could explore this</span><br>
<span>… we could set up guidelines about when to set up a discussion and when to open an issue</span></p>
<p><cite>AvneeshSingh:</cite> multiple channels of communication can be trouble<br>
<span>… we already have internal reading list, public reading list</span></p>
<p><cite>SueNeu:</cite> What is the functional difference between a discussion and an issue in GH that makes discussions a better idea?</p>
<p><cite>wendyreid:</cite> discussions are designed to be a bit more like talking to people<br>
<span>… the accessibility group uses them, responses can be threaded</span><br>
<span>… one of the problems can be hard to follow comments when they overlap</span><br>
<span>… it makes for a better flow of discussions</span></p>
<p><cite>mgarrish:</cite> the threaded replies don't always work the way they are intended<br>
<span>… we would want some guidance for people about when to open discussions and how to respond</span><br>
<span>… if you don't have a specific problem you want addressed in the spec it makes sense to set up a discussion even if the discussion threading isn't perfect</span></p>
<p><cite>wendyreid:</cite> I'll do some research on GH about discussions<br>
<span>… this might be more geared toward opensource projects</span><br>
<span>… there is some merit giving people space to ask a question that isn't a direct issue</span></p>

<hr /></details>
